lib/js-api/src/api/gs-core-rest-api/model/hold.ts

```diff
@@ -15,10 +15,8 @@
  * limitations under the License.
  */
 
-export interface Hold {
+import { HoldBody } from './holdBody';
+
+export interface Hold extends HoldBody {
```

Comment:
I'm not sure I like this since this approach makes `reason` here required but it will always be null when hold is fetched from api which feels weird

Reply:
align to create separate interface for `getNodeAssignedHolds`. In other places `reason` is mandatory